app/scripts/services/colors.coffee

```diff
@@ -28,9 +28,9 @@ angular.module('edudashAppSrv').service 'colorSrv', ->
   pinOff: (colour) ->
     style =
       color: '#fff'
-      fillOpacity: 1
-      opacity: 0.75
-      weight: 1
+      fillOpacity: 0.96
+      opacity: 0.95
+      weight: 2
```

Comment:
Hey @esoliani  I just changed it back to 1. I did it to try to match the appearance in the old dashboard because of a request from the client but I reverted for now.  :)